gradle.properties-example

```diff
@@ -1,2 +1,3 @@
+org.gradle.jvmargs=-Xmx1536m
```

Comment:
FluxC's default value. I don't have any preferences about the memory, the default one happened to be so small and CI was having OOM on `Metaspace`.

Reply:
Praise (❤️): I agree, nice touch! 🙇